x/shield/types/events.go

```diff
@@ -14,20 +14,23 @@ const (
 	EventTypeCreateCompensation     = "create_compensation"
 	EventTypeWithdrawReimbursement  = "withdraw_reimbursement"
 
-	AttributeKeyShield             = "shield"
-	AttributeKeyDeposit            = "deposit"
-	AttributeKeySponsor            = "sponsor"
-	AttributeKeyPoolID             = "pool_id"
-	AttributeKeyAdditionalTime     = "additional_time"
-	AttributeKeyTimeOfCoverage     = "time_of_coverage"
-	AttributeKeyBlocksOfCoverage   = "blocks_of_coverage"
-	AttributeKeyCollateral         = "collateral"
-	AttributeKeyDenom              = "denom"
-	AttributeKeyToAddr             = "to_address"
-	AttributeKeyAccountAddress     = "account_address"
-	AttributeKeyAmount             = "amount"
-	AttributeKeyPurchaseID         = "purchase_id"
-	AttributeKeyCompensationAmount = "compensation_amount"
-	AttributeKeyBeneficiary        = "beneficiary"
-	AttributeValueCategory         = ModuleName
+	AttributeKeyShield              = "shield"
+	AttributeKeyDeposit             = "deposit"
+	AttributeKeySponsor             = "sponsor"
+	AttributeKeyPoolID              = "pool_id"
+	AttributeKeyAdditionalTime      = "additional_time"
+	AttributeKeyTimeOfCoverage      = "time_of_coverage"
+	AttributeKeyBlocksOfCoverage    = "blocks_of_coverage"
+	AttributeKeyCollateral          = "collateral"
+	AttributeKeyDenom               = "denom"
+	AttributeKeyToAddr              = "to_address"
+	AttributeKeyAccountAddress      = "account_address"
+	AttributeKeyAmount              = "amount"
```

Comment:
What's the logic for the changed ordering? If no good reason, let's append instead of mutating existing ordering.

Reply:
Ok I will append. But most of these lines changed because of lint.